docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -461,7 +486,12 @@ class Sidebar extends React.Component<any, any> {
       disabled: true,
     }
 
-    const withComponents = withPrototypes.concat(componentMenuItem).concat(menuItemsByType[0].items)
+    const shownMenuItems = query ? this.renderSearchItems() : menuItemsByType[0].items
+
+    const withComponents = withPrototypes
+      .concat(searchInput)
```

Comment:
Input cannot be in a menu - you would need to split it to multiple menus or move the input to somewhere else.

Reply:
we should include this explanation as part of the Do's and Dont's for the Menu